devito/dle/parallelizer.py

```diff
@@ -22,7 +22,7 @@ def ncores():
 class NThreads(Constant):
 
     def __new__(cls, **kwargs):
-        return super(NThreads, cls).__new__(cls, name='nthreads', dtype=np.int32,
+        return super(NThreads, cls).__new__(cls, name=kwargs['name'], dtype=np.int32,
```

Comment:
name is always in kwargs?

Reply:
yes, it has to be, and this is enforcing that